modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -293,7 +293,12 @@ Optional: You can bond multiple SR-IOV network interfaces to a dual port NIC int
 
 On each node, you must perform the following tasks:
 
+ifndef::installing-ibm-power[]
 . Create the SR-IOV virtual functions (VFs) following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/managing-virtual-devices_configuring-and-managing-virtualization#managing-sr-iov-devices_managing-virtual-devices[Managing SR-IOV devices]. Follow the procedure in the "Attaching SR-IOV networking devices to virtual machines" section.
+endif::installing-ibm-power[]
```

Comment:
I'm not an expert on conditionals but are lines 298 and 299 necessary? Couldn't you open the conditional at line 296 and close it at line 301?

Reply:
@JoeAldinger updated the code as below:
```
ifndef::installing-ibm-power[]
. Create the SR-IOV virtual functions (VFs) following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/managing-virtual-devices_configuring-and-managing-virtualization#managing-sr-iov-devices_managing-virtual-devices[Managing SR-IOV devices]. Follow the procedure in the "Attaching SR-IOV networking devices to virtual machines" section.
endif::installing-ibm-power[]
ifdef::installing-ibm-power[]
. Create the SR-IOV virtual functions (VFs).
endif::installing-ibm-power[]
```
Thank you for pointing out. The second part is power only.